scripts/build/windows.bat

```diff
@@ -212,10 +212,7 @@ call %electron_packager% . %application_name%^
  --out=%output_build_directory%
 
 set package_output=%output_build_directory%\%package_name%
-
-if not "%arch%"=="%electron_arch%" (
-  move %output_build_directory%\Etcher-win32-%electron_arch% %package_output%
-)
```

Comment:
I think this if-test should be restored, with the line inside the if changed to:
```
  move %output_build_directory%\Etcher-win32-%electron_arch% %output_build_directory%\Etcher-win32-%arch%
```

because if %arch% is "x86" then %electron_arch% will be "ia32" https://github.com/WasabiFan/etcher/blob/47d0594019cf5d775c3f9ee118e099a853be4baf/scripts/build/windows.bat#L127